config/routes.rb

```diff
@@ -15,6 +15,7 @@
     get "/", to: "dashboard#show"
     get "/items", to: "items#index"
     get "/orders/:id", to: "orders#show"
+    patch '/items/:id/update_status', to: 'items#update_status'
```

Comment:
@Jonathan-M-Wilson this could just be `items#update` bc that's a CRUD action (keeping it RESTful) - in theory, this could be used to update any part of the item, not just `status`, and it will be once we implement editing an item.

Reply:
I did it again! Changes have been made to help keep it RESTful.